daylight_saving.md

```diff
@@ -0,0 +1,59 @@
+NTFS Timezones and Daylight saving
+=================
+
+## Handling time-zones
+
+The `network_timezone` field in file networks.txt sets the time zone for all `stop_times` of `trips` of the `network`. 
+
+If a feed covers multiple time zones, a specific time zone information can be specified on a `stop_point` outside the `network_timezone` in the `stop_timezone` field. Set this optional field in file `stops.txt` (for stops with location_type = 0). If not set, the timezone of the network is used. Ensure that trip travel times remain accurate and consistent by providing departure and arrival times based on the `network_timezone` for stops that are out of that timezone.
```

Comment:
I'm not sure `outside the 'network_timezone'` is useful since you already use _specified_ so we understand it's gonna be more specific. You can probably remove it.